### PR TITLE
Setting explicit properties

### PR DESCRIPTION
**Pull request by andreasohlund**
Adjust doco and upgrade guide to mention the removal of `.ConfigureProperty` and `.InitializeHandlerProperty`

### CODE REVIEW

File: nservicebus/containers/property-injection.md [hunk 1]
```diff
@@ -9,21 +9,20 @@ redirects:
  - nservicebus/property-injection-in-handlers
 ---
 
-When using the NServiceBus built-in container it is possible to control property injection.
+NServiceBus will automatically enable property injection known types across the supported IOC Containers. Use the lambda overload of `.ConfigureComponent` to get full control over the injected properties if needed.
```
Comment (SimonCropp) [inline nservicebus/containers/property-injection.md]: i would use the word Func or Action rather than lambda

Comment (SimonCropp) [inline nservicebus/containers/property-injection.md]: > IOC Containers

to 

> Containers

and make it a link

File: nservicebus/containers/property-injection.md [hunk 2]
```diff
@@ -9,21 +9,20 @@ redirects:
...
+Setting the properties would be done as follows:
 
+snippet: ConfigurePropertyInjectionForHandler
 
 ## A Handler/Saga specific API
 
-From Versions 5.2 and above a new, more explicit, API has been introduced that specifically targets Handlers and Sagas.
+Versions 5.2 supported a new, more explicit, API that specifically targets Handlers and Sagas.
+
+NOTE: This api has been obsoleted in Version 6
```
Comment (SimonCropp) [inline nservicebus/containers/property-injection.md]: obsoleted with error or warning? do we need to be explicit?

**Comment by SimonCropp** [comment]
@andreasohlund needs a rebase. also r u still working on this?

**Comment by andreasohlund** [comment]
I'll get to it today

**Comment by andreasohlund** [comment]
Rebased and updated based on @SimonCropp 's comments. Also removed WIP since it doesn't depend on the core pull being merged

**Comment by SimonCropp** [comment]
merged manually since the rebase was stuffed

**Pull request closed without merging**